src/components/form/form.js

```diff
@@ -15,6 +16,31 @@ export default class Form extends React.Component {
     };
   }
 
+  // Loops through the usersPath using the next key in the array as a reference to generate the original
+  // Question and Answer from the Wayfinder content data object
+  // Returns array of Question and Answer objects (eliminating 'continue' answers)
+  generateUserResponseDataFromPath = () => {
```

Comment:
I think there is a bug here. Where the legal orders are concerned, the question and answer data is not updated correctly if the user goes back as they have the same goto in wayfinder data file. 

Also it might be worth preventing the user from changing their choices after they submit the form as this could potentially result in duplicated forms for GP. I will write an issue about this as only the bug needs to be addressed before merge.

Reply:
I've updated wayfinderData.js so that SGO/CAO answers now have separate keys - this was causing the bug. 

We will also need a unit test for this function to catch edge cases.